pydash/pydash_app/dashboard/endpoint.py

```diff
@@ -61,3 +61,42 @@ def aggregated_data(self):
         :return: A dict containing aggregated data points.
         """
         return self._aggregator_group.fetch_aggregator({}).as_dict()
+
+    def aggregated_data_daterange(self, start_date, end_date, granularity):
+        """
+        Returns the aggregated data on this dashboard over the specified daterange.
+        :param start_date: A datetime object that is treated as the inclusive lower bound of the daterange.
+        :param end_date: A datetime object that is treated as the inclusive upper bound of the daterange.
+        :param granularity: A string denoting the granularity of the daterange.
+        :return: A dictionary with all aggregated statistics and their values.
+        """
+        return self._aggregator_group.fetch_aggregator_inclusive_daterange({}, start_date, end_date,
+                                                                           granularity).as_dict()
+
+    def statistic(self, statistic, filters):
+        """
+
```

Comment:
Missing documentation?

Reply:
Fair point, I'll get to writing the documentation.